flink-streaming-java/src/test/java/org/apache/flink/streaming/api/operators/async/AsyncWaitOperatorTest.java

```diff
@@ -100,6 +102,9 @@
 public class AsyncWaitOperatorTest extends TestLogger {
 	private static final long TIMEOUT = 1000L;
 
+	@Rule
+	public Timeout timeoutRule = new Timeout(10, TimeUnit.SECONDS);
```

Comment:
Why we need this change? 
Some previous tests were configured separately as 2 seconds, but now we unify it for 10 seconds for all the cases.

Reply:
These timeouts are just a safe-guard whenever we break stuff. All tests should always complete without timeout. But it will be especially useful on Travis as we do not need to wait for Travis to timeout to get a test result.

Instead of just annotating the two tests that just failed (as we did in the past), I'd annotate the whole suite with the junit rule, which makes it more readable. Imho @1u0 even started with that rule and I just copied him ;).